### DISCUSSION TITLE
Enhance @info font size output

### DISCUSSION THREAD

**Discussion by jgardona** [fonsp/Pluto.jl#3111] [Pluto development]
Hello, good morning,
Im working on windows 11 with pluto, and notice that the @info macro output generates a lesser font sized answer as we can see below:

![image](https://github.com/user-attachments/assets/cebdbab3-f1f8-4c67-b8ed-61b1242103be)

Outputting an array will got the things better for visualization:

![image](https://github.com/user-attachments/assets/924a8b61-5e7c-4ece-ade9-f1f537b72c35)

May we enhance a litte bit, the first output? I already got the chrome page in zoom and adjust the dpi scale, but its not enought.

**Comment by fonsp** [comment]
Are you using Symbolics.jl?
